src/main/scala/beam/sim/BeamServices.scala

```diff
@@ -111,6 +111,16 @@ class BeamServicesImpl @Inject()(val injector: Injector) extends BeamServices {
     iterationNumber += 1
     Metrics.iterationNumber = iterationNumber
   }
+
+  def maybeScaleTransit(vehicleTypes: TrieMap[Id[BeamVehicleType], BeamVehicleType]): TrieMap[Id[BeamVehicleType], BeamVehicleType] = {
+    Option(beamConfig.beam.agentsim.tuning.transitCapacity) match {
```

Comment:
This won't work as you think, as it's a Double, so even if empty it would be 0.0 - however it comes with a 1.0 default anyway:

    beam.agentsim.tuning.transitCapacity = "double | 1.0"

You're better option would be to change the type in the template to be optional from the start.